### PR TITLE
Update chart from upstream

### PR DESCRIPTION
**Pull request by taylorbot**
This PR was created by the `update-chart` automated workflow.

**:warning: Make sure all tests have passed before merging.**

### CODE REVIEW

**Comment by Strigix** [comment]
Hey Zach, could you review and approve? 
I'll put the question to update kube-linter at Honeybadger (at least, that's the team I think has the ownership).

**Review by stone-z — COMMENTED** [review]
Lots of interesting changes coming in here. The PR itself looks fine but please test extensively and update the changelog

File: helm/falco/charts/falco/BREAKING-CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    --reuse-values \
+    --set falcoctl.artifact.install.enabled=false \
+    --set falcoctl.artifact.follow.enabled=false
+```
+This way you will upgrade Falco to `v0.34.0`.
+
+**NOTE**: The new version of Falco itself, installed by the chart, does not introduce breaking changes. You can port your previous Falco configuration to the new `values.yaml` by copy-pasting it.
+
+
+### Falcoctl support
```
[the remark this answers is not in the document]
Comment (Strigix) [reply helm/falco/charts/falco/BREAKING-CHANGES.md:26]: _"how does this change the rules we override? We have added some macros and modified some rules in the config repo -- are those overrides still honored?"_ 

So these are still being honored, as I've seen the (GS) rules in the logs. Also the rules are still added to the rules.d folder.

I'm just a bit stuck on the part underneath, does this mean we'll have to change we the way we update? WDYT? @stone-z

[the remark this answers is not in the document]
Comment (stone-z) [reply helm/falco/charts/falco/BREAKING-CHANGES.md:26]: We talked. We think this is fine for now. Let's try it as is and watch out for breakage

**Review by stone-z — DISMISSED** [review]

**Comment by Strigix** [comment]
update PR is failing because it sees no reason to (already up-to-date/clean) 
So you can just review and approve.
Then I can merge & release.

**Comment by fhielpos** [comment]
The `Create Update Chart PR` fails because it tries to create a PR that already exists (this one). You can ignore it 😄 

@Strigix  Can you please update this line first?
https://github.com/giantswarm/falco-app/blob/cdced586d655ce4bab672d9408294b53bf679a56/helm/falco/Chart.yaml#L2C14-L2C14

It should be the same `appVersion` than the falco chart.

**Review by fhielpos — APPROVED** [review]
Nice. The `appVersion` is not updated automatically, we need to automate it but it's a bit tricky.

**Pull request merged**